guides/split_testing.md

```diff
@@ -106,3 +106,7 @@ The experience the user sees is determined by the traffic split percentage you s
 When a split test is active, the XDN will automatically set an `xdn_destination` cookie to the name
 of the chosen destination. You can use this value in the browser to report the split test experience assignment to
 analytics.
+
+## Compatibility with A/B testing tools
+
+Moovweb XDN split testing routes traffic at the edge based on a variety of criteria. It does not identify user cohorts (although it can split on cohorts identified by annother tool) or report business metrics about A/B tests since there are many great tools for that. We recommend you utlize an A/B testing tool that supports server-side integration such as Monetate, Optimizely, Adobe Test, Google Experiments, or Visual Web Optimizer. These tools will set a cookie or header that can be used to split traffic using the appropriate criteria described above.
```

Comment:
```suggestion
Moovweb XDN split testing routes traffic at the edge based on a variety of criteria. It does not identify user cohorts (although it can split on cohorts identified by another tool) or report business metrics about A/B tests since there are many great tools for that. We recommend you utilize an A/B testing tool that supports server-side integration such as Monetate, Optimizely, Adobe Test, Google Experiments, or Visual Web Optimizer. These tools will set a cookie or header that can be used to split traffic using the appropriate criteria described above.
```